amoro-ams/src/main/java/org/apache/amoro/server/AmoroManagementConf.java

```diff
@@ -342,16 +343,16 @@ public class AmoroManagementConf {
           .defaultValue(1000L)
           .withDescription("Max wait time before getting a connection timeout.");
 
-  public static final ConfigOption<Long> OPTIMIZER_HB_TIMEOUT =
+  public static final ConfigOption<Duration> OPTIMIZER_HB_TIMEOUT =
       ConfigOptions.key("optimizer.heart-beat-timeout")
-          .longType()
-          .defaultValue(60000L)
+          .durationType()
+          .defaultValue(Duration.ofMinutes(1))
           .withDescription("Timeout duration for Optimizer heartbeat.");
 
-  public static final ConfigOption<Long> OPTIMIZER_TASK_ACK_TIMEOUT =
+  public static final ConfigOption<Duration> OPTIMIZER_TASK_ACK_TIMEOUT =
       ConfigOptions.key("optimizer.task-ack-timeout")
-          .longType()
-          .defaultValue(30000L)
+          .durationType()
+          .defaultValue(Duration.ofSeconds(30))
```

Comment:
If a user sets this conf to `60000` before this commit, and he/she needs to adjust the config to `60`, am I understanding right? if it is true, can we still use millisecond as the unit in the current commit?